taxi/views.py

```diff
@@ -86,4 +87,31 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
-    queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+    queryset = Driver.objects.prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+
+
+class DriverLicenseUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverLicenseDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+def assign_unassign_to_car(request, *args, **kwargs):
+    car_pk = kwargs["pk"]
+    users_cars = request.user.cars
+    get_object_or_404(Car, id=car_pk)
```

Comment:
just extract `car` and `driver` into variables with this method. and then use something like that `if driver in car.drivers.all()`.
Would be much more easier to read and you will get rid of redundant lines

Reply:
Fixed all